library/core/src/mem/maybe_uninit.rs

```diff
@@ -485,9 +485,10 @@ impl<T> MaybeUninit<T> {
     ///     }
     /// }
     /// ```
-    #[stable(feature = "maybe_uninit_write", since = "1.55.0")]
-    #[rustc_const_unstable(feature = "const_maybe_uninit_write", issue = "63567")]
     #[inline(always)]
+    #[stable(feature = "maybe_uninit_write", since = "1.55.0")]
+    #[cfg_attr(bootstrap, rustc_allow_const_fn_unstable(const_mut_refs))]
```

Comment:
It should be possible to remove the `rustc_allow_const_fn_unstable` now the stage0 compiler has been updated.